CRM/Core/Form/Task/PDFLetterCommon.php

```diff
@@ -241,12 +243,12 @@ public static function processTemplate(&$formValues) {
       if (!empty($formValues['bind_format']) && $formValues['format_id']) {
         $messageTemplate['pdf_format_id'] = $formValues['format_id'];
       }
-      if (!empty($formValues['saveTemplate']) && $formValues['saveTemplate']) {
+      if (!empty($formValues['saveTemplate'])) {
         $messageTemplate['msg_title'] = $formValues['saveTemplateName'];
```

Comment:
Agree that it was added for e_notices, but when I first looked at this I was thinking maybe one was meant to be saveTemplateName since that's what the block actually acts on, but it's ok if you assume that the formRule above about it will never change.

Anyway all the changes look fine and I did a light r-run and works ok.

Reply:
yeah - it's a question not of 'was there a mistake' but 'which mistake was made' :-)